packages/apps/docs/src/components/Layout/Full/index.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import { fullLayoutFrontMatter } from '../__fixtures__/frontmatter';
```

Comment:
layout onsmall screen is different as in browser.
header seems to have disapeared
![image](https://github.com/kadena-community/kadena.js/assets/4015521/dea3cb03-7cb4-4b1c-bde5-71a7cf67d600)


in my browser:

![image](https://github.com/kadena-community/kadena.js/assets/4015521/bc455c9e-b3d5-4e50-a382-c0713c13624d)

Reply:
As I mentioned, as part of storybook didn't implement complete page! this storybook contains only the layout part. So, it will not have header component.